osmnames/prepare_data/set_polygon_types.sql

```diff
@@ -8,3 +8,9 @@ WHERE mapping.country_code = polygon.country_code
   AND mapping.admin_level = polygon.admin_level
   AND polygon.admin_level IS NOT NULL
   AND polygon.type = 'administrative';
+
+UPDATE osm_polygon AS polygon
+  SET type = override.type
+FROM admin_level_type_mapping_override AS override
+WHERE override.country_code = polygon.country_code
```

Comment:
isn't the `country_code` redundant when comparing the `osm_id` ?

Reply:
Probably. I left it in just in case there was some optimization with country code and to protect against the possibility that the osm_id from libpostal is incorrect and belongs to a different country.